frontend/src/services.js

```diff
@@ -623,6 +623,12 @@ CoreServices.factory('Pulse', ['$resource', '$cookies', function($resource, $coo
             params: { pulseId: '@pulseId' },
             headers: { 'X-CSRFToken': () => $cookies.csrftoken },
         },
+        test: {
+            url: '/api/pulse/test',
+            method: 'POST',
+            params: { pulseId: '@id' },
```

Comment:
This endpoint is for unsaved pulses so we can test unsaved changes. I don't think it makes sense to include the id in the URL (we can also remove the `pulseId` param)